hack/build.sh

```diff
@@ -42,5 +42,11 @@ if (echo "${TAGS}" | grep -q 'libvirt')
 then
 	export CGO_ENABLED=1
 fi
+if test "$(go env GOARCH)" = "arm64"
```

Comment:
Is this change needed when compiling **on** arm64 or **targeting** arm64?

Reply:
Targeting arm64.